### PR TITLE
Fix reactive table filters

### PR DESCRIPTION
**Pull request by s7dhansh**
Your suggested solution works !

### CODE REVIEW

File: lib/filter.js [hunk 1]
```diff
@@ -180,6 +181,7 @@ if (Meteor.isClient) {
     return _.map(filterIds, function (filterId) {
       if (_.isUndefined(reactiveTableFilters[filterId])) {
         reactiveTableFilters[filterId] = new ReactiveTable.Filter(filterId);
+        return "";
```
Comment (aslagle) [inline lib/filter.js]: Does it still work without this line? This will reintroduce the bug that 0.8.29 was fixing.

Comment (s7dhansh) [inline lib/filter.js]: yes it does. updating it.

**Comment by aslagle** [comment]
Published version 0.8.31 with this change :)

**Comment by s7dhansh** [comment]
thank you!

**Pull request merged**